docs/howto/query.md

```diff
@@ -60,12 +65,14 @@ if err != nil {
 }
 ```
 
-> See more [`sqlair.Prepare`](https://pkg.go.dev/github.com/canonical/sqlair#Prepare)
+> See more:
```

Comment:
Looks like too much space. Please double-check.

Reply:
If they're in the same paragraph they are rendered as one block, I think this should be fine